CHANGES/1031.contrib.rst

```diff
@@ -0,0 +1 @@
+Update mypy, remove unused ignores and casts
```

Comment:
Don't use imperative mood. Change notes have different style. Optionally, add a byline. Add a trailing period to the sentence. Use proper capitalization for MyPy.